pkg/cmd/cli/cmd/deploy.go

```diff
@@ -284,14 +286,17 @@ func (o *DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer
 		return err
 	}
 	if len(deployments.Items) == 0 {
-		fmt.Fprintln(out, "no deployments found to cancel")
+		fmt.Fprintf(out, "there have been no deployments for %s/%s\n", config.Namespace, config.Name)
```

Comment:
Oh, it's because this is used with an error: prefix.  Capitalizing after the colon looks wrong